### PR TITLE
Add WNFS session

### PR DESCRIPTION
**Pull request by expede**

### CODE REVIEW

File: events/b_1_ipfs-impls.toml [hunk 1]
```diff
@@ -97,9 +97,9 @@ description=""
 
 [[timeslots]]
 startTime="14:15"
-speakers=["@expede?"]
+speakers=["@expede","@matheus23"]
 title="WNFS"
-description=""
+description="The WebNative File System adds versioning, recursive encryption, and eventual consistency on top of IPFS"
```
Comment (matheus23) [inline events/b_1_ipfs-impls.toml]: "Recursive encryption" to me sounds too much like [re-encrypting ciphertexts](https://security.stackexchange.com/questions/13254/where-can-we-learn-more-about-recursive-encryption-schemes), which isn't that exciting. Maybe something like "cryptree-based encryption"?

```suggestion
description="The WebNative File System adds versioning, cryptree-based encryption, and eventual consistency on top of IPFS"
```

File: events/b_1_ipfs-impls.toml [hunk 2]
```diff
@@ -97,9 +97,9 @@ description=""
 
 [[timeslots]]
 startTime="14:15"
-speakers=["@expede?"]
+speakers=["@expede","@matheus23"]
 title="WNFS"
```
Comment (matheus23) [inline events/b_1_ipfs-impls.toml]: Also, similar to the UCAN session title, maybe we expand the acronym once?

```suggestion
title="WebNative File System (WNFS)"
```

**Comment by expede** [comment]
@matheus23 committed changes!

Also: I appreciate that you're respecting my PR, but in this case feel free to make changes and submit upstream 👍 I can PR against that copy if I really want something else in there 😄

**Comment by matheus23** [comment]
I submitted this to upstream: https://github.com/ipfs-shipyard/ipfs-thing-2022/pull/42

**Comment by matheus23** [comment]
This got merged :+1:

**Pull request closed without merging**